docs/advanced-entity-analytics/behavioral-detection-use-cases.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[behavioral-detection-use-cases]]
+= Behavioral detection use cases
+
+Behavioral use cases detect potential insider and external threats based on observed user actions and host activities. These use cases employ a threat-centric approach to flag deviations or suspicious activities by analyzing patterns, anomalies, and context enrichment. 
+
+Elastic builds behavioral detection use cases on its foundational SIEM detection capabilities, leveraging {ml} algorithms and AI to enable proactive threat detection and hunting.
```

Comment:
This reads like we're saying Elastic builds use cases, which doesn't sound right to me. We build a feature or capability that _has_ use cases. Also I think its redundant to say machine learning and AI, feels buzzword-y. 
```suggestion
{elastic-sec}'s behavioral detection feature builds on its foundational SIEM detection capabilities, leveraging {ml} algorithms to enable proactive threat detection and hunting.
```

Reply:
I think you're right, but I'll defer to @SourinPaul to confirm.

Reply:
Agree with this suggestion too ^.